.rubocop_todo.yml

```diff
@@ -500,11 +464,6 @@ Style/RegexpLiteral:
 Style/RescueEnsureAlignment:
   Enabled: false
 
-# Offense count: 46
```

Comment:
I agree, this is a recurring problem and shouldn't simply be disabled. I agree somewhat with the other inline comments, and care more about the work this puts on maintainers than the results of a poll.